### DISCUSSION TITLE
Dissecting the Web3 Narrative

### DISCUSSION THREAD

**Discussion by AndreasS2501** [rufuspollock/awesome-crypto-critique#175] [General]
Hello fellow life-itself people :)

Today I came across this piece which really offers something by digging deeper into what "Decentralization" (but not only) actually can/should mean in the context of web3. 

The Article points out a few interesting things where many current web3 projects fall short of "promises":

- Identity and Reputation
- Data Empowerment
- Organizational Innovation

Interesting concept perspective on subsidiarity vs redundancy

https://www.wired.com/story/web3-blockchain-decentralization-governance/

In spring I explored a similar direction with this  Keynote  - Decentralization, Autonomy and Organization - by jaya klara brekke
My notes for that can be found here: https://andreass.gitlab.io/2022/KeynoteNotes.html

**Comment by rufuspollock** [comment]
@AndreasS2501 thanks for sharing. One option here is to write up short notes on that piece and add a wiki page under `/notes/` in the repo?

**Reply by AndreasS2501** [reply]
Once I'm through with the piece I can do that. I think the notes from  jaya klara brekke's keynote are also meaningful in this context.

**Reply by AndreasS2501** [reply]
I just checked with my zettelkasten and I have 5 Zettel with the tags #web3 and #narrative . There are some pieces like : 
https://the-crypto-syllabus.com/web3-a-map-in-search-of-territory/
Who are quite pessimistic. I think its quite extraordinary interesting how the discussion about a narrative like web3 brings us closer  to a point of view where it allows us to speak about how we craft cultural narratives and what could we improve on the process in general but also on the current example web3. 
For example Morozov mostly identifies web3 as a narrative driven by venture capitalists and certainly at least since late 2021 there is a strong influence coming from that side. 
But many of the , what I would call infrastructure projects, like ethereum are not (entirely? - to what degree)  VC driven. There are indeed strange dynamic loops going on. But as I see Projects like Gitcoin or Filecoin focusing on the public goods discussion. I do see the potential for these projects and in turn web3 to become meaningful. 
This is also acknowledged by the wired article refering to the subject of quadratic funding which gitcoin uses. 
I think that was one of your ponderings @rufuspollock in this context #72 .
Dissecting further how many web3 projects articulate their relationship to solarpunk and lunarpunk is also relevant in this context.

**Reply by rufuspollock** [reply]
@AndreasS2501 suggest a start is a pull request for one new note and we can review. Do you want to submit the jaya klara notes first (if you have them already done)? /cc @EilidhRoss1

**Reply by AndreasS2501** [reply]
Hi @rufuspollock I have created a pull request for that.

**Comment by AndreasS2501** [comment]
I finally found the time to write some notes on the wired piece they look like this: 
# The Web3 Decentralization Debate Is Focused on the Wrong Question

- #tags: #web3 #narrative #perspective #decentralization #cooperatives #trust #article
- https://www.wired.com/story/web3-blockchain-decentralization-governance/
- dissecting fianlly the decentralization "myth"
- "Thus, there are (soft) limits to the degree of centralization and to the decentralization feasible in a functional system."
- "The "decentralization" currently achieved by most canonical crypto projects, such as Bitcoin, is what we call "distributed redundancy": global, open, consensus-based storage in many locations of a common, homogeneous data set. Distributed redundancy depends on three factors"
- 1. Maximally removing data from social context. (All interactions are boiled down to transactions as recorded in the ledger, with outside context unable to be represented in the technical architecture.) (my remark: the context problem is real!)
- 2. Aiming for universalized solutions. (A focus on "global" applicability requires all solutions to be applicable in all contexts.)
- 3. Relying on global consensus and redundant verification accessed using fungible resources. (Decisionmaking mechanisms are limited by tokens or computational puzzles; people with more financial resources have more of these.)
- "In sharp contrast to these principles, the type of decentralization that we believe is desirable, subsidiarity, focuses on:
- 1. Keeping data as close as possible to the social context of creation.
- 2. A plurality of solutions linked and integrated through coordinated mechanisms of federation and interoperability.
- 3. Leveraging and extending relationships of online and offline trust and institutions.
- The examples of "networks of networks":
- 1. The ActivityPub standard for social networks and associated applications such as Mastodon.
- 2. A range of research projects from Ink and Switch, including an architecture for “local-first” computing and designs for interoperability across diverse programs.
- 3. A range of socially local identity systems, such as Spritely, BrightID, and BackChannel.
- 4. Federated learning and broader privacy-preserving machine learning.
- 5. Mesh networks.
- 6. Data collaboratives, cooperatives, and trusts.
- 7. Wikipedia and wiki-based content structure more generally.
- 8. Community content moderation systems, such as Reddit.
- 9. Community-first cloud computing, file storage, and time-sharing.
- **Identity and Reputation**
- Promise: "Web3 promises to liberate identity and reputation from the control of a few large tech companies, allowing for “self-sovereign” identities for communication, transaction, and governance."
- Redundancy: multiple accounts on Pseudonymous ledgers
- "Solutions that take redundancy as a prior focus on universal, decontextualized unique cryptographic identifiers. Removing context leads to reliance on “universally secure” identifiers based on clean/universal features like biometrics, which often raise at least as many concerns as the centralized protocols they replace."
- Subsidary: "**Trust** is the basic building block for identity, and most relevant interactions that turn to identity mechanisms for proof or verification are far more about **relationships** (status as employee, citizen, student, platform contributor) than they are about universal identification.
- **Data Empowerment**
- Promise: "Web3 claims to let data creators “own,” and likely profit from, their data, while protecting their privacy."
- Redundancy: "The typical vision of data ownership focuses on a private property conception of data in “personal data stores” that can be freely transacted and that interface with the "market" via a DeFi structure."
- Subsidary: data cooperatives (where to put data union? https://www.dataunion.app/ saw them in ocean discord)
- **Organizational Innovation**
- Promise: "Web3 and specifically the Distributed Autonomous Organization (DAO) movement promise flexible, lightweight, accountable organizations and the empowerment of peer-to-peer, holocratic communities."
- Redundancy: largely failed ( trent and mayn other agree), 51% attacks, scope problems etc.
- Subsidary: because ( also) of problems in the DAO space people took inspiration from the real world cooperatives platfrom cooperatism https://platform.coop/ https://pol.is/home ( democracy app ?) gitcoin
- "**Web3 has provoked an important discussion about decentralization.** However, it is time to harness this energy to achieve what is best about decentralization: **subsidiarity**, not redundancy—a network of networks, not a ledger."
